src/lib/arch/aarch64/InstructionMetadata.cc

```diff
@@ -983,6 +1346,11 @@ void InstructionMetadata::revertAliasing() {
         }
         return;
       }
+      if (opcode == Opcode::AArch64_NOTv16i8 ||
+          opcode == Opcode::AArch64_NOTv8i8) {
+        // mvn vd.t, vn.t; alias for : not vd.t, vn.t
```

Comment:
Blank entry for same reason as AArch64_MUL_ZPmZ_{B,D,H,S} instructions?

Reply:
Blank entry was for a legitimate alias, however operands were identical so nothing to alter between the instructions. I will add an additional comment.

Reply:
Comment added : `// Blank entry was for a legitimate alias, however operands were identical so nothing to alter between the instructions.`

Reply:
Sounds good, want me to add it in my commits

Reply:
Please